swift/ql/lib/codeql/swift/generated/PlatformVersionAvailabilitySpec.qll

```diff
@@ -8,7 +8,8 @@ module Generated {
    * An availability spec based on platform and version, for example `macOS 12` or `watchOS 14`
    */
   class PlatformVersionAvailabilitySpec extends Synth::TPlatformVersionAvailabilitySpec,
-    AvailabilitySpec {
+    AvailabilitySpec
+  {
```

Comment:
A number of these changes are in generated files.  Changes need to be made at the source (`schema.py` or `qlgen.py`???) then the files regenerated with `bazel run //swift/codegen` I think.

Reply:
I'm not sure that's necessary? It looks like the relevant codegen has some hook to format its output, and I guess that makes the most sense anyway - it would be brittle to rely on a code generator generating correctly formatted code, when the generator could simply run the autoformatter.

Reply:
Awesome, thanks.